### PR TITLE
Page to display all accounts

### PR DESCRIPTION
**Pull request by matthieu-gendreau**

### CODE REVIEW

**Comment by tdeo** [comment]
Can we use the gem instead of dropping JS assets ? https://github.com/bjevanchiu/bootstrap-table-rails

**Comment by matthieu-gendreau** [comment]
I tried to use the gem but the export didn't work with it.
I think we need a vendor folder to put the external js and css assets in.

**Comment by tdeo** [comment]
Nevermind then for the gem.

This doesn't allow to filter right ? Like status = ... AND promo = ... AND solde < 0 ?

**Comment by matthieu-gendreau** [comment]
I should implement it with https://github.com/lukaskral/bootstrap-table-filter.
But it is such a mess I didn't manage to make it work.

**Comment by matthieu-gendreau** [comment]
Ohoh I was about to push the rest

**Comment by tdeo** [comment]
What rest ? Filtering ?

**Comment by matthieu-gendreau** [comment]
Yes. And for that I had to modify the bootstrap-table file

**Comment by tdeo** [comment]
Can't you just try to add the extra javascript ? I've put some in vendor/assets/javascripts already

**Comment by matthieu-gendreau** [comment]
I will try

**Comment by tdeo** [comment]
Tu as trouve une librairie ? Et qui supporte de faire solde < 0 ?

**Comment by tdeo** [comment]
sinon tu pousse une autre branche a partir de ton premier commit aussi.

**Comment by matthieu-gendreau** [comment]
Oui tu as raison j'ai poussé une autre branche pour l'instant.
J'ai utilisé https://github.com/lukaskral/bootstrap-table-filter mais ça marchait pas du tout de base j'ai dû fouiller dans le code pour le modifier et le faire marcher.

File: vendor/assets/javascripts/bootstrap-table-filter-extension.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
```
Comment (tdeo) [inline vendor/assets/javascripts/bootstrap-table-filter-extension.js]: This file is in the gem https://github.com/bjevanchiu/bootstrap-table-rails/blob/master/vendor/assets/javascripts/extensions/bootstrap-table-filter.js: You can require the gem version with `//= require 'extensions/bootstrap-table-filter'`. Only difference is `throw 'Error: searchCallback not defined'`, which shouldn't matter too much?

Comment (matthieu-gendreau) [reply vendor/assets/javascripts/bootstrap-table-filter-extension.js]: Exact we can delete it.

File: app/assets/stylesheets/account.scss [hunk 2]
```diff
@@ -31,3 +31,8 @@ form.form-inline {
 .pagination {
   margin: 0px 0px 20px;
 }
+
+// bootstrap-table-filter
+.fixed-table-toolbar .dropdown-menu {
```
Comment (tdeo) [inline app/assets/stylesheets/account.scss]: Can that go in bootstrap_ext.scss ?

**Comment by tdeo** [comment]
Nice !

**Pull request merged**